tests/test_pourbaix_diagram.py

```diff
@@ -0,0 +1,356 @@
+from __future__ import annotations
+import pymatgen
+import pyEQL
+import multiprocessing
+from unittest import TestCase
+
+import matplotlib.pyplot as plt
+import numpy as np
+from monty.serialization import dumpfn, loadfn
+from pytest import approx
+
+from pyEQL.Pourbaix.pourbaix_diagram import (
+    IonEntry,
+    MultiEntry,
+    PourbaixDiagram,
+    PourbaixEntry,
+    PourbaixPlotter,
+    ion_or_solid_comp_object,
+)
```

Comment:
looks like you've got a casing error in the import statement. `Pourbaix` should be `pourbaix`